gym_roboy/envs/msj_env.py

```diff
@@ -15,7 +15,7 @@ class MsjEnv(gym.GoalEnv):
 
     _max_tendon_speed = 0.02  # cm/s
     _JOINT_ANGLE_BOUNDS = np.ones(MsjRobotState.DIM_JOINT_ANGLE) * np.pi
-    _JOINT_VEL_BOUNDS = np.ones(MsjRobotState.DIM_JOINT_ANGLE) * np.inf
+    _JOINT_VEL_BOUNDS = np.ones(MsjRobotState.DIM_JOINT_ANGLE) * np.pi / 6  # 30 deg/sec
```

Comment:
why in capital letters?

Reply:
because we would like to indicate that it is a constant?

Reply:
Yes